src/database/sqlite/sqlite_context.c

```diff
@@ -13,6 +13,13 @@ const char *database_context_config[] = {
     "last_time_t INT NOT NULL, deleted INT NOT NULL, "
     "family TEXT, PRIMARY KEY (host_id, id))",
 
+    "CREATE TABLE IF NOT EXISTS context_metadata_cleanup (id INTEGER PRIMARY KEY, host_id BLOB, context TEXT NOT NULL, date_created INT, "
```

Comment:
Considering the upcoming queries, the date_created field should not be `null` either.